backend/Dockerfile

```diff
@@ -1,6 +1,19 @@
-FROM python:3.9-slim-buster as base
+FROM python:3.9-slim as base

```

Comment:
that's quite a big change : debian 10 -> debian 12, it's a good idea but it's worth testing if it doesn't break anything.

I would prefer you write explicitly "python:3.9-slim-bookworm" because it fixes the version of debian it uses, whereas the base image for "python:3.9-slim" can vary over time.